spec/spec_helper.rb

```diff
@@ -13,7 +13,7 @@
 # it.
 
 require 'simplecov'
-# SimpleCov.minimum_coverage 87
+SimpleCov.minimum_coverage 87
```

Comment:
Did you mean to uncomment this as part of the PR? Is 87% really where we are at the moment (a little low for a new app)